### DISCUSSION TITLE
problem w/ Copy of QuickStart Deepgram - Speaker Labeling Notebook.ipynb - could use a little help

### DISCUSSION THREAD

**Discussion by gliason** [deepgram/community#267] [General help]
Hi,
The notebook is working but only around 7 minutes from a 40 minute mp3 make it to the .json file and text. I'm trying to transcribe 1 hour interviews. I know nothing about code. I followed the instructions so the notebook works to that degree.
What do I need to change?
I appreciate any help.
George

**Comment by jjmaldonis** [comment]
Hey @gliason I'm assuming you're using this link: https://colab.research.google.com/drive/1Jo3q5ZE6MzYJl8UOqc1Xne2DcxTvFIdt?usp=sharing#scrollTo=e7CgWdImOs-E

Have you uploaded your own audio files into the folder/directory?

Have you made any changes to the code at all?

Coding is one of those things where the devil is in the details so it's hard to debug without knowing much more information.
